be/src/exec/sorting/sort_column.cpp

```diff
@@ -412,6 +466,12 @@ Status sort_and_tie_column(const std::atomic<bool>& cancel, ColumnPtr& column, c
     return column->accept(&column_sorter);
 }
 
+static Status sort_and_tie_column(const std::atomic<bool>& cancel, const Column* column, const SortDesc& sort_desc,
```

Comment:
why we don't have to fill_null_with_default here ?
```
if (column->is_nullable() && !column->is_constant()) {
        ColumnHelper::as_column<NullableColumn>(column)->fill_null_with_default();
    }
```

Reply:
Fixed.